### PR TITLE
22w19a misc

### PR DESCRIPTION
**Pull request by apple502j**
Contains Realms 32bit warning screen refactor, animation, `ChunkBuilder` stuff (partially), mineshaft generation, and leftover from recent snapshots (loom + jigsaw).

### CODE REVIEW

File: mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping [hunk 1]
```diff
@@ -142,9 +142,16 @@ CLASS net/minecraft/class_846 net/minecraft/client/render/chunk/ChunkBuilder
 				ARG 2 cameraY
 				ARG 3 cameraZ
 			METHOD method_23087 addBlockEntity (Lnet/minecraft/class_846$class_851$class_4578$class_7435;Lnet/minecraft/class_2586;)V
+				ARG 1 renderData
+				ARG 2 blockEntity
 			METHOD method_23619 (Lnet/minecraft/class_846$class_849;Ljava/util/List;Ljava/lang/Throwable;)Lnet/minecraft/class_846$class_4690;
 				ARG 2 results
 				ARG 3 throwable
+			CLASS class_7435 RenderData
+				FIELD field_39079 blockEntitiesRenderedOutsideBoundingBoxes Ljava/util/List;
```
Comment (enbrain) [inline mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping]:
```suggestion
				FIELD field_39079 noCullingBlockEntities Ljava/util/List;
```
To be consistent with `WorldRenderer#noCullingBlockEntities`.

File: mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping [hunk 2]
```diff
@@ -142,9 +142,16 @@ CLASS net/minecraft/class_846 net/minecraft/client/render/chunk/ChunkBuilder
...
+				FIELD field_39080 blockEntities Ljava/util/List;
+				FIELD field_39082 chunkOcclusionData Lnet/minecraft/class_854;
+				FIELD field_39083 bufferBuilderState Lnet/minecraft/class_287$class_5594;
```
Comment (enbrain) [inline mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping]:
```suggestion
				FIELD field_39083 sortState Lnet/minecraft/class_287$class_5594;
```

File: mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping [hunk 3]
```diff
@@ -142,9 +142,16 @@ CLASS net/minecraft/class_846 net/minecraft/client/render/chunk/ChunkBuilder
...
+				ARG 1 renderData
+				ARG 2 blockEntity
 			METHOD method_23619 (Lnet/minecraft/class_846$class_849;Ljava/util/List;Ljava/lang/Throwable;)Lnet/minecraft/class_846$class_4690;
 				ARG 2 results
 				ARG 3 throwable
+			CLASS class_7435 RenderData
+				FIELD field_39079 noCullingBlockEntities Ljava/util/List;
+				FIELD field_39080 blockEntities Ljava/util/List;
+				FIELD field_39082 chunkOcclusionData Lnet/minecraft/class_854;
+				FIELD field_39083 sortState Lnet/minecraft/class_287$class_5594;
```
Comment (enbrain) [inline mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping]:
```suggestion
				FIELD field_39083 translucenySortingData Lnet/minecraft/class_287$class_5594;
```

Comment (haykam821) [reply mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping]: That suggestion has a typo:

```suggestion
				FIELD field_39083 translucencySortingData Lnet/minecraft/class_287$class_5594;
```

**Review by modmuss50 — APPROVED** [review]

**Review by enbrain — APPROVED** [review]

**Pull request merged**